custom/languages/en_UK.json

```diff
@@ -219,7 +219,7 @@
     "admin/internal": "Internal",
     "admin/external": "External",
     "admin/plugin": "Plugin",
-    "admin/external_query_help": "If the default internal server query does not work, try another option.",
+    "admin/query_type_help": "If the default internal server query does not work, try another option.",
```

Comment:
Would we want to change this to recommend plugin query instead of internal query?

Reply:
If internal query is available (and it is on most hosting providers), external query is slower and less reliable by comparison